core/src/main/java/me/lokka30/treasury/plugin/core/config/messaging/Messages.java

```diff
@@ -112,6 +112,30 @@
                     "%prefix% For a list of available subcommands, try '&b/%label% help&7'."
             );
 
+            @Comment("Placeholders: %prefix%, %label%")
+            @Key("subcommands.downloadLatest.invalid-usage")
+            private String downloadLatestInvalidUsage = "%prefix% Invalid usage, try '&b/%label% downloadLatest&7'";
+
+            @Comment("Placeholders: %prefix%")
+            @Key("subcommands.downloadLatest.started")
+            private String downloadLatestDownloadStarted = "%prefix% &aDownload process started.";
+
+            @Comment("Placeholders: %prefix%")
+            @Key("subcommands.downloadLatest.error-occurred")
+            private String downloadLatestIOException = "%prefix% &cUnable to retrieve information/download jar. See console for more details.";
+
+            @Comment("Placeholders: %prefix%")
+            @Key("subcommands.downloadLatest.couldnt-parse")
+            private String downloadLatestCouldntParse = "%prefix% &cSomething went wrong whilst parsing latest build download data. Please notify a Treasury developer ASAP!";
+
+            @Comment("Placeholders: %prefix%")
+            @Key("subcommands.downloadLatest.already-latest")
+            private String downloadLatestAlreadyLatest = "%prefix% &eYou are already running the latest version of Treasury!";
+
+            @Comment("Placeholders: %prefix%, %time%")
+            @Key("subcommands.downloadLatest.success")
+            private String downloadLatestSuccess = "%prefix% &aSuccessfully downloaded the latest Treasury jar (took %time%ms). Please restart your server in order for it to take effect.";
```

Comment:
does not follow standard color scheme:

use `&7` instead of `&a`
`%time%ms` should be highlighted using `&b`